### ISSUE TITLE
Single quote parsing

### ISSUE THREAD

**Issue by round** [lczerniawski/Unused-CSS-Finder#1]
Doesn't seem to detect classes wrapped in single quotes e.g. `<div class='className'>`

**Comment by lczerniawski** [comment]
Yes, that's the case, thanks for creating the issue! This is now fixed in v1.0.2.

**Comment by round** [comment]
I can confirm it works – thank you so much for fixing (and making) this!

**Comment by lczerniawski** [comment]
No problem, I'm glad you like it! 😄
